packages/twenty-front/src/pages/settings/developers/SettingsDevelopersApiKeys.tsx

```diff
@@ -0,0 +1,92 @@
+import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import styled from '@emotion/styled';
+
+import { CoreObjectNameSingular } from '@/object-metadata/types/CoreObjectNameSingular';
+import { useFindManyRecords } from '@/object-record/hooks/useFindManyRecords';
+import { SettingsApiKeysFieldItemTableRow } from '@/settings/developers/components/SettingsApiKeysFieldItemTableRow';
+import { ApiFieldItem } from '@/settings/developers/types/ApiFieldItem';
+import { formatExpirations } from '@/settings/developers/utils/format-expiration';
+import { IconPlus } from '@/ui/display/icon';
+import { H2Title } from '@/ui/display/typography/components/H2Title';
+import { Button } from '@/ui/input/button/components/Button';
+import { Section } from '@/ui/layout/section/components/Section';
+import { Table } from '@/ui/layout/table/components/Table';
+import { TableBody } from '@/ui/layout/table/components/TableBody';
+import { TableHeader } from '@/ui/layout/table/components/TableHeader';
+import { TableRow } from '@/ui/layout/table/components/TableRow';
+
+const StyledDiv = styled.div`
+  display: flex;
+  justify-content: flex-end;
+  padding-top: ${({ theme }) => theme.spacing(2)};
+`;
+
+const StyledTableRow = styled(TableRow)`
+  grid-template-columns: 312px 132px 68px;
+`;
+
+const StyledTableBody = styled(TableBody)`
+  border-bottom: 1px solid ${({ theme }) => theme.border.color.light};
+`;
+
+export const SettingsDevelopersApiKeys = () => {
+  const navigate = useNavigate();
+
+  const [apiKeys, setApiKeys] = useState<Array<ApiFieldItem>>([]);
+
+  useFindManyRecords({
+    objectNameSingular: CoreObjectNameSingular.ApiKey,
+    filter: { revokedAt: { is: 'NULL' } },
+    orderBy: {},
+    onCompleted: (data) => {
+      setApiKeys(
+        formatExpirations(
+          data.edges.map((apiKey) => ({
+            id: apiKey.node.id,
+            name: apiKey.node.name,
+            expiresAt: apiKey.node.expiresAt,
+          })),
+        ),
+      );
+    },
+  });
+
+  return (
```

Comment:
there is an issue when no apiKeys or webhook are listed 
![image](https://github.com/twentyhq/twenty/assets/29927851/2be19e75-e482-40ca-824b-e3db45b8e5ef)
![image](https://github.com/twentyhq/twenty/assets/29927851/874a7cd9-cecb-4c61-a0a3-1dd61cacbeb1)
Think you should remove the last line in the Table when there is no apiKey. Same for webhook. Would look like 
![image](https://github.com/twentyhq/twenty/assets/29927851/302a66bd-78bf-4ce2-8859-14a9a4937acc)

Seen with @Bonapara